pyoos/parsers/ioos/describe_sensor.py

```diff
@@ -8,7 +10,7 @@
 class IoosDescribeSensor(object):
 
     def __new__(cls, element):
-        if isinstance(element, str):
+        if isinstance(element, bytes):
```

Comment:
Yep. This PR is old and I probably missed many of those. This should be `six.string_types` everywhere. I will check for those.